src/core/components/choice-card/index.tsx

```diff
@@ -37,6 +38,7 @@ interface ChoiceCardGroupProps extends Props {
 	supporting?: string
 	multi?: boolean
 	error?: string
+	columns?: number
```

Comment:
It's unlikely this component will ever need an unlimited number of columns. Being strict about the number of allowable columns will let us [avoid writing dynamic styles](https://www.theguardian.design/2a1e5182b/p/77c9d9-contributing/t/902350)

```tsx
type Columns = 2 | 3 | 4 | 5 | undefined
```